### ISSUE TITLE
Split timezone for start/end (nice to have)

### ISSUE THREAD

**Issue by yourealwaysbe** [xsoh/Etar-Calendar#84]
Would it be possible to allow different timezones for the start and end of events?  This would  be useful for adding flights to the calendar, where the timezone you start from differs from the timezone you finish.  (At the moment i do this by syncing with an ics file and manually editing it...)

**Comment by rparkins999** [comment]
Yes, I would like this too, for the same reason. It could be implemented by adjusting the event start time if you change the time zone. For example I start a flight event taking off from London at 12:00 (GMT), and then modify the time zone in the event to New York (GMT-5). Etar should modify the start time to 0700 (the same instant in GMT-5). Then I can put in my landing time as given by the airline 15:00 (GMT-5).

**Comment by petrhejna** [comment]
I miss this feature so much. Every time I travel I have to calculate timezones by hand I live in fear I miscalculate and miss the flight 👻 

**I put 25$ bounty in this one. (Paid in BTC once the feature gets merged to master.)**

**Comment by AidanJHMurphy** [comment]
+1 to this. I'm always on edge at the airport.

**Comment by rparkins999** [comment]
I implemented this some time ago in my version (https://github.com/rparkins999/Etar-Calendar). Each event has a separately displayed and settable time zone for start time and end time. I can't offer a pull request because my version has too many other changes.

**Comment by rparkins999** [comment]
The underlying Android Calendar database has fields for both event start timezone and event end timezone, but the original Etar/Group/Etar-Calendar model didn't. So the first step is to add the event end timezone to the model. Then you have to arrange to make it  displayable and editable. If I remember correctly (it was a long while ago that I forked from it) Etar/Group/Etar-Calendar had two different event views, one for just looking at it and another for editing it. I thought this wasn't a particularly good UI (at least for me) and had too much redundant code. So I amalgamated them into a single EditEventView(.java) using vertical scrolling to put everything into a single page. That file in rparkins99/Etar-Calendar contains the code to display and edit separate time zones for start and end. It's basically a copy, modify, and paste job on the original code to edit just one time zone. The original change is in commit 30195834e0e87f65a836b1578d4c8f13b032f000, but there were quite a few bug fixes later.
